pynxtools/dataconverter/readers/stm/stm_file_parser.py

```diff
@@ -0,0 +1,383 @@
+"""
+    A parse stm file in nexus generated definition template.
```

Comment:
Please try to force you to make as clean as possible docstrings, I know also that I frequently make typo and grammar mistakes but we should try our best it really adds to code readability and interpretability, I once had a colleague who wrote comments in the code complaining about the functionality of certain code portions in the mother language that inside some Fortran code :) this told me to better when I am old, in case you ask: no I was not that person complaining the source code in my mother tongue :)